src/main/java/Task.java

```diff
@@ -0,0 +1,40 @@
+public class Task {
+
+    protected String description;
+    protected boolean isTaskDone;
```

Comment:
I like the naming of this boolean variable, it is clear what it is representing.